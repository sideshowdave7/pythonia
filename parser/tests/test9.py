x = 3 if 1 == 2 else 4
y = 3 if False else 8 - 7/2
global a, b
b = 3
def foo():
    global a
    a = 3
    b = 2
print(a)
print(b)

