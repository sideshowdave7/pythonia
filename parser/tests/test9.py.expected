(program
 (= (x) (if (comparison 1 ("==" 2)) 3 4))
 (= (y) (if False 3 (arith 8 ("-" (term 7 ("/" 2))))))
 (global a b)
 (= (b) 3)
 (def (foo) (suite (global a) (= (a) 3) (= (b) 2)))
 (expr (indexed print (called a)))
 (expr (indexed print (called b))))

